spec/system/cookies_spec.rb

```diff
@@ -19,29 +19,29 @@
 
       it 'visitor can click to accept analytics cookies' do
         click_on 'Accept analytics cookies'
-        expect(get_me_the_cookie('track_analytics')[:value]).to eq 'Yes'
+        expect(get_me_the_cookie('track_analytics')[:value]).to eq 'true'
```

Comment:
Looks like this spec is ripe for 
```ruby
get_me_the_cookie('track_analytics')[:value]
```
getting moved into a `let` block

Reply:
good point